cmd/bridge/config/config.toml

```diff
@@ -46,7 +46,6 @@
 
 [P2P]
     Port = "10010"
-    Seed = ""
```

Comment:
Why is this line deleted ?

Reply:
in mx-chain-go, the rc/v1.4.0 got a new (better) method of generating deterministic p2p keys by using an external p2pKey.pem file